ui/seek_bar.js

```diff
@@ -200,6 +205,8 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
   /**
    * Called by Controls on a timer to update the state of the seek bar.
```

Comment:
These docs should go to the interface, so someone can understand the interface without referring to this implementation.

Reply:
Done